apps/src/javalab/Backpack.jsx

```diff
@@ -0,0 +1,160 @@
+import React, {Component} from 'react';
+import Radium from 'radium';
+import color from '@cdo/apps/util/color';
+import onClickOutside from 'react-onclickoutside';
+
+const placeholderFiles = [
+  'MyClass.java',
+  'MyPainter.java',
+  'NeighborhoodMain.java'
+];
+
+const styles = {
+  dropdown: {
+    position: 'absolute',
+    top: 30,
+    backgroundColor: color.darkest_gray,
+    color: color.light_gray,
+    zIndex: 20,
+    paddingTop: 5,
+    paddingBottom: 5,
+    borderRadius: 2,
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center'
+  },
+  buttonStyles: {
+    cursor: 'pointer',
+    float: 'right',
+    backgroundColor: color.light_purple,
+    marginTop: 3,
```

Comment:
nit: these 4 margin styles can be combined into 1 like so: `margin: '3px 3px 3px 0px'` (order is top, right, bottom, left)